.circleci/config.yml

```diff
@@ -156,15 +156,15 @@ commands:
                 command: |
                   $Env:GOOGLE_APPLICATION_CREDENTIALS = $Env:CI_GCS_PATH
                   setx /m GOOGLE_APPLICATION_CREDENTIALS "$Env:GOOGLE_APPLICATION_CREDENTIALS"
-                  python3 -m pytest --cov-report=xml --cov=./ --benchmark-autosave
+                  python3 -m pytest --cov-report=xml --cov=./ --benchmark-enable --benchmark-autosave
```

Comment:
> Forcibly enable benchmarks. Use this option to override –benchmark-disable (in case you have it in pytest configuration).

Where are we disabling benchmarks that we need this to re-enable them?

This feels out-of-scope.

Reply:
> in .circleci/config.yaml run pytest & pytest-benchmark separately (with --benchmark-skip & --benchmark-only flags)

it is out of scope, but it is necessary since the benchmarks take a while to run. `pytest .` will only run tests, `pytest . --benchmark-only` will run benchmarks